lang/values.md

```diff
@@ -313,17 +313,17 @@ For `Value`, we lift the order on byte lists to relate `Bytes`s, and otherwise r
 impl PartialOrd for Value {
     fn le(self, other: Self) -> bool {
         match (self, other) {
-            (Int(i1), Int(i2)) =>
+            (Value::Int(i1), Value::Int(i2)) =>
                 i1 == i2,
-            (Bool(b1), Bool(b2)) =>
+            (Value::Bool(b1), Value::Bool(b2)) =>
                 b1 == b2,
-            (Ptr(p1), Ptr(p2)) =>
+            (Value::Ptr(p1), Value::Ptr(p2)) =>
                 p1 <= p2,
-            (Tuple(vals1), Tuple(vals2)) =>
+            (Value::Tuple(vals1), Value::Tuple(vals2)) =>
                 vals1 <= vals2,
-            (Variant { idx: idx1, data: data1 }, Variant { idx: idx2, data: data2 }) =>
+            (Value::Variant { idx: idx1, data: data1 }, Value::Variant { idx: idx2, data: data2 }) =>
                 idx == idx1 && data1 <= data2,
-            (Union(chunks1), Union(chunks2)) => chunks1 <= chunks2,
+            (Value::Union(chunks1), Value::Union(chunks2)) => chunks1 <= chunks2,
```

Comment:
This should also have a `use`.

(Not sure where I put the threshold but when there are 12 uses in a single `match` it's definitely worth a `use`. Maybe the threshold is around 5.)